core/src/main/codegen/templates/Parser.jj

```diff
@@ -5379,6 +5381,34 @@ SqlCall JsonLengthFunctionCall() :
     }
 }
 
+SqlCall JsonRemoveFunctionCall() :
+{
+    final List<SqlNode> elements = new ArrayList<SqlNode>();
+    final SqlNode[] otherArgs = new SqlNode[1];
+    SqlNode e;
+    final Span span;
+}
+{
+    <JSON_REMOVE> { span = span(); }
+    <LPAREN> [
+        e = JsonValueExpression(true) {
+            otherArgs[0] = e;
+        }
+        (
+            <COMMA>
+            e = JsonValueExpression(false) {
```

Comment:
Per the doc[1] I think we can just use regular expression here. The argument is not a JSON in general.

[1] https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-remove

Reply:
Well, I can parse it as a `Expression(ExprContext.ACCEPT_NON_QUERY)` and pass it through `ctx.read(element.tostring ())! = null` to determine whether the deletion condition is met.